ern-orchestrator/src/generateContainerForRunner.ts

```diff
@@ -48,6 +48,7 @@ export async function generateContainerForRunner(
       runLocalCompositeGen({
         baseComposite,
         jsApiImpls,
+        metroExtraNodeModules: extra?.compositeGenerator?.metroExtraNodeModules,
```

Comment:
https://github.com/electrode-io/electrode-native/blob/master/ern-orchestrator/test/generateContainerForRunner-test.ts
Test in

Reply:
OK, I added a test. At some point it would be a good idea to reevaluate how much value some of these "intermediate level" unit tests (just checking if other internal methods were called/passing through arguments). I see a lot of repetitive code that does not actually check for any correctness.